sequencing-server/src/lib/batchLoaders/simulatedActivityBatchLoader.ts

```diff
@@ -228,6 +228,8 @@ function convertType(value: any, schema: Schema): any {
       return Temporal.Duration.from(parse(value).toISOString());
     case SchemaTypes.Boolean:
       return value;
+    case SchemaTypes.Path:
```

Comment:
Not blocking (and technically not this PR), but is there a reason we don't use fall-through/`case A || B` for the Real, Boolean, Path, and String cases? It's fine if the answer's style, it just stood out that we have four cases that all are the same behavior.

Reply:
I was going to modify this, but with the `assertUnreachable` it actually breaks if we go that route now because it has parsing that all the cases have been hit.